epicJWTforBackend.js

```diff
@@ -1,58 +1,60 @@
-// Add to code templates https://raw.githubusercontent.com/kjur/jsrsasign/master/jsrsasign-all-min.js
-// Tested with version 10.7.0 
-
-// IMPORTANT In the code template you added you must add this near the top because this library can be used with browsers.
-// var navigator = {}  // fake the fact we have no browser
-// var window = {}     //
-
-// On Epic dev portal, you will need to upload your public key, their documentation is clear 
-// on generating your required rsa private and public keys and retreive your ClientID
-
-// Transformer code:
-
-// time
-const iat = Math.floor(Date.now() / 1000);
-
-// Set up the JWT header and payload
-var header =
-    {
+/**
+ * Generates a JWT for Epic FHIR server authentication.
+ * Requires https://raw.githubusercontent.com/kjur/jsrsasign/master/jsrsasign-all-min.js as a codetemplate
+ * Tested with version 10.7.0
+ * IMPORTANT In the code template you added you must add this near the top because this library can be used with browsers.
```

Comment:
What errors or problems are thrown if one doesn't do this?

IOW if the user gets a FooBar error then that is a signal that they need to do this. Leave breadcrumbs

Reply:
Comment updated.

Reply:
@jonbartels I don't think even ChatGPT is smart enough to guess what errors this code could throw. How would the type conversion from Java to JavaScript impact the error type? =) And I hate the Github emoji's can never remember them.

Reply:
We're both old enough to remember when we used ASCII and not emojis.

The inline code comment leads me to believe that the author only found that important requirement through trial and error. If a specific error was encountered when the code is not added near the top, then that error should be listed in the documentation. My comment is not asking someone to guess, it is asking them to show.